web-main/src/app/components/pages/cons-page/components/con-body/components/msg-sender/msg-sender.component.ts

```diff
@@ -7,6 +12,31 @@ import * as ngCore from '@angular/core';
   selector: 'app-msg-sender'
 })
 export class MsgSenderComponent {
-  
+  private readonly _store = ngCore.inject(ngrxStore.Store)
+  private readonly _conApiService = ngCore.inject(ConApiService)
```

Comment:
THIS IS BIG ANTI PATTERN, you should **NEVER** inject API service into component, we do everything through ngrx. Take a look at this [image](https://ngrx.io/generated/images/guide/store/state-management-lifecycle.png)